build/tasks/aria-supported.js

```diff
@@ -0,0 +1,82 @@
+/*eslint-env node */
+'use strict';
+
+const { roles, aria: props } = require('aria-query');
+const mdTable = require('markdown-table');
+
+module.exports = function (grunt) {
+	grunt.registerMultiTask(
+		'aria-supported',
+		'Task for generating a diff of supported aria roles and properties.',
+		function () {
+			const entry = this.data.entry;
+			const destFile = this.data.destFile;
+			const listType = this.data.listType.toLowerCase();
+
+			/**
+			 * `axe` has to be dynamically required at this stage, as `axe` does not exist until grunt task `build:uglify` is complete, and hence cannot be required at the top of the file.
+			 */
+			const axe = require('../../axe');
+
+			/**
+			 * As `aria-query` roles map, does not list all aria attributes in its props,
+			 * the below reduce function aims to concatanate and unique the below two, 
+			 * - list from props with in roles map
+			 * - list from aria map
+			 * 
+			 * @return {Map} `aQaria` - This gives a composite list of aria attributes, which is later used to diff against axe-core supported attributes.
```

Comment:
This JSDoc-style comment is confusing. The `Array.from().map()` returns an Array, not a Map. What is returning a Map here?

Maybe replace all of this with:

```js
// Coerce aria-query's Maps to unique Arrays
```